nemo/collections/common/parts/adapter_modules.py

```diff
@@ -61,6 +61,10 @@ def get_default_strategy_config(self) -> 'dataclass':
         """
         return adapter_mixin_strategies.ResidualAddAdapterStrategyConfig()
 
+    def unfreeze(self,):
```

Comment:
Should be reverted. This is a mixin class - and if you inherit a NeuralModule - it will have a multiple inheritance diamond type inheritance. 

If you need unfreeze then you should inherit NeuralModule instead of torch.nn.Module (its almost exactly the same thing with utility methods)

Reply:
fair point, but what about adapters like `MultiHeadAttentionAdapter` which do not inherit NeuralModel this call from `adapter_mixins` will fail `module.adapter_layer[name].unfreeze()`